src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java

```diff
@@ -151,6 +152,21 @@ private String extractArgumentValue(String argsString,
             NattyParser nattyParser = NattyParser.getInstance();
             value = nattyParser.parseNLPDate(value);
         }
+
+        //@@author A0124591H
+        if (prefix.equals(PREFIX_BLOCK)) {
+            // Parse date
+            NattyParser nattyParser = NattyParser.getInstance();
+            String[] splitValue = value.split(" to ");
```

Comment:
`"to"` should be a `final String` . This is currently similar to a _magic number_